specification/configuration.md

```diff
@@ -158,14 +158,15 @@ For all use-cases that support environment variables (e.g. applications and
 serverless), it MUST be possible to configure an Instrumentation Library
 instance using the following environment variables:
 
-| Name (default value)                            | Description                                                                     |
-| :------------------------------------:          | :--------------------------------------------------------:                      |
-| `SPLUNK_ACCESS_TOKEN` ()                        | Access token added to exported data. [1]                                        |
-| `SPLUNK_METRICS_ENDPOINT` ()                    | Endpoint for metrics data ingest.                                               |
-| `SPLUNK_PROFILER_ENABLED` (`false`)             | Whether profiling is enabled. [2]                                               |
-| `SPLUNK_PROFILER_LOGS_ENDPOINT` ()              | Where profiling data is sent. By default `OTLP_EXPORTER_OTLP_ENDPOINT` is used. |
-| `SPLUNK_REALM` (`none`)                         | Which realm to send exported data. [3]                                          |
-| `SPLUNK_TRACE_RESPONSE_HEADER_ENABLED` (`true`) | Whether `Server-Timing` header is added to HTTP responses. [4]                  |
+| Name                                   | Default | Description                                                     |
+|----------------------------------------|---------|-----------------------------------------------------------------|
+| `SPLUNK_ACCESS_TOKEN`                  | none    | Access token added to exported data. [1]                        |
+| `SPLUNK_METRICS_ENDPOINT`              | none    | Endpoint for metrics data ingest.                               |
+| `SPLUNK_PROFILER_CALL_STACK_INTERVAL`  | 10s     | Interval at which call stacks are sampled                       |
+| `SPLUNK_PROFILER_ENABLED`              | false   | Whether profiling is enabled. [2]                               |
+| `SPLUNK_PROFILER_LOGS_ENDPOINT`        | *       | Where profiling data is sent. Defaults to the value in `OTLP_EXPORTER_OTLP_ENDPOINT`  |
+| `SPLUNK_REALM`                         | `none`  | Which realm to send exported data. [3]                          |
```

Comment:
What's the difference between a ticked none and a not ticked none? Perhaps blank default should not list anything in this column? Or I guess actual values can be ticked?

Reply:
I agree, this is confusing. I didn't actually know the answer and just brought it forward the way it was. Lemme go find the answer...

Reply:
`none` is a valid and expected default value for the `SPLUNK_REALM`. It was discussed here: https://github.com/signalfx/gdi-specification/pull/148

The other values should be left blank

Reply:
Thanks for the clarification, @pellared I'll get it changed.